tests/integration/test_pip_packages.py

```diff
@@ -27,3 +27,61 @@ def test_failing_pip_local_path(test_env):
         f"#{completed_response.id}: Request failed correctly, but with unexpected message: "
         f"{completed_response.data['state_reason']}. Expected message was: {error_msg}"
     )
+
+
+def test_failing_pip_invalid_req_path(test_env):
```

Comment:
I see your point, we have these simple integration tests in OSBS as well. We're using pytest markers to have a different level of testing based on our requirements.
* Just a simple quick test to check open PRs.
* Nightly test which tests 90% of scenarios.
* And Full regression test just for our releases where we include also slow/long test scenarios like building base images for our releases.

I would vote for having a single integration test that targets a malformed repository.

Reply:
I was checkgin the code with Sumin, and it seems I was wrong on my assumptions: a repository without a `requirements.txt` file is a valid scenario, so the test I proposed is not valid. :sweat_smile: 

It seems that you are in favor of having more integrationt tests, so in this case, I believe this could be left as @sumincho22 implemented it.

Reply:
@lkolacek @taylormadore Please let me know what you guys think, so I know if I should revert it back to the two original integration tests I had.